sortinghat/command.py

```diff
@@ -57,20 +60,22 @@ def display(self, template, **kwargs):
 
         t = env.get_template(template)
         s = t.render(**kwargs)
-
+        logger.info(s)
+        # this is left to not break the tests
         sys.stdout.write(s)
 
     def error(self, msg):
         s = "Error: %s\n" % msg
-        sys.stderr.write(s)
+        logger.error(s)
```

Comment:
I wouldn't change this. We have to make a difference between those messages related to the internal work of the tool (log messages) and those messages generated as an output for the user (which is this case).

Reply:
ok @sduenas, I'm going to revert this change and update the PR